packages/documentation-site/patternfly-docs/content/get-started/develop.md

```diff
@@ -5,154 +5,134 @@ section: get-started
 ---
 import './get-started.css';
 
+## Getting started 
 
-Before you begin, check out our [overview of PatternFly](/get-started/about) to get acquainted with the basic elements of the design system.
+In order to develop with PatternFly, you will need to:
 
-**Before you begin, install a package manager:**
+1. [Learn about the PatternFly design system](/get-started/about) to get acquainted with all of its elements.
 
-* [Use npm](https://nodejs.org/en/download)
-* [Use Yarn](https://yarnpkg.com/en/docs/getting-started)
+1. [Learn about our design token system](/tokens/about-tokens), which enables you to build interfaces with consistent PatternFly styling, including colors, shadows, spacing, and more.
```

Comment:
This link returns a 404 for me, but maybe that's just because things are still in flight?

Reply:
oh yeah since the current url is https://staging-v6.patternfly.org/tokens/about-tokens, but since this pr merges into the v6 branch I think what I have should work once this doc goes live with v6?